src/continuation/continuation_afam.jl

```diff
@@ -0,0 +1,107 @@
+"""
+    AttractorsContinueAndMatch(mapper, matcher [, seeding])
```

Comment:
The `seeding` argument is optional? In the type implementation it seems mandatory to declare seeding

Reply:
OK, I got it. The concrete implementations are still to be implemented. There will be several implementations depending on the type of mapper and matcher.